target/linux/ath79/files/drivers/spi/spi-rb4xx.c

```diff
@@ -0,0 +1,430 @@
+/*
```

Comment:
This is a direct copy of https://github.com/openwrt/openwrt/blob/master/target/linux/ar71xx/files/drivers/spi/spi-rb4xx.c with a few changes (annotated below)